src/V1Meta.js

```diff
@@ -54,4 +54,10 @@ export default class V1Meta {
         const headers = createHeaderObj(this.authHeader);
         return Promise.resolve(this.postFn(url, null, headers));
     }
+
+    queryDefinition(assetType) {
+        const url = `${this.urls.meta()}/${assetType}`;
+        const headers = createHeaderObj(this.authHeader);
+        return Promise.resolve(this.postFn(url, null, headers));
```

Comment:
Why enforce JSON when we don't have to? Maybe someone doesn't want to consume json... it is a concern of the api to determine what it supports to return not the sdk..?